server/meca/routes.js

```diff
@@ -37,20 +37,34 @@ module.exports = app => {
 
     const updateStatus = () => Manuscript.updateStatus(manuscriptId, status)
 
-    const sendEmail = () => mailer.send({
-      to: config.get('meca.email.recipient'),
-      from: config.get('meca.email.sender'),
-      subject: `${config.get('meca.email.subjectPrefix')}MECA import failed`,
-      text: `
+    const auditlog = () =>
+      new AuditLog({
+        action: 'UPDATED',
+        objectId: manuscriptId,
+        objectType: 'ejp.id',
+        value: body.id,
+      }).save()
+
+    const sendEmail = () =>
+      mailer
+        .send({
+          to: config.get('meca.email.recipient'),
+          from: config.get('meca.email.sender'),
+          subject: `${config.get(
+            'meca.email.subjectPrefix',
+          )}MECA import failed`,
+          text: `
 EJP failed to import MECA package.
 Manuscript ID: ${manuscriptId}
       `,
-    }).catch((err) => {
-      throw new Error(`MECA import failure email failed to send. ${err}`)
-    })
+        })
+        .catch(err => {
+          throw new Error(`MECA import failure email failed to send. ${err}`)
+        })
 
     updateStatus()
       .then(() => {
+        auditlog()
```

Comment:
`auditLog()` returns a promise, so you'll want to wrap this within a separate `.then`